runtime/doc/ui.txt

```diff
@@ -789,6 +789,7 @@ must handle.
 	kind
 	    Name indicating the message kind:
 		"" (empty)	Unknown (consider a |feature-request|)
+		"bufwrite"	|:write| message
```

Comment:
we have "list_cmd" and "search_cmd' already, should this be "write_cmd" ? Or is this triggered by other ways of writing a buffer?

Reply:
IDK; "list_cmd" contains output from one of various _commands_ that list something, "search_cmd" contains the actual search _command_ itself, not a message, so they're all a bit different. But yeah this message comes from the :write _command_ so the _cmd suffix could make sense.